cw_evm/lib/evm_chain_wallet.dart

```diff
@@ -323,6 +322,8 @@ abstract class EVMChainWalletBase
       contractAddress:
           transactionCurrency is Erc20Token ? transactionCurrency.contractAddress : null,
       data: hexOpReturnMemo,
```

Comment:
i.e remove it from here as well, it shouldn't be needed

Reply:
Swaps for ETH don't work without the memo/data parameter.